package.json

```diff
@@ -37,7 +37,7 @@
     "react-dropzone": "^14.2.2",
     "react-helmet": "^6.1.0",
     "react-number-format": "^5.0.1",
-    "react-router-dom": "^6.3.0",
+    "react-router-dom": "^6.2.1",
```

Comment:
Had to downgraded to version `6.2.1` because `6.3.0` removed the `block` function.